.envSample

```diff
@@ -3,3 +3,7 @@ SECRET=
 NODE_ENV=
 TEST_DATABASE_URL=
 DATABASE_URL=
+DB_USER=postgres
+DB_PASSWORD=juninho1
+DB_NAME=weTravel
+DB_HOST=127.0.0.1
```

Comment:
Remove this values, we only need the variable names such as;
```DB_USER
DB_PASSWORD
DB_NAME
DB_HOST```

Reply:
ok noted